examples/intro_clean.ipynb

```diff
@@ -1124,22 +1145,22 @@
     "        )\n",
     "\n",
     "    def generate_steps(self, agent, dialog, llm_stream: LLMStream):\n",
-    "        o = llm_stream.get_output()\n",
-    "        if o.content:\n",
-    "            yield SetNextNode(next_node=0)\n",
-    "            yield AssistantStep(content=o.content)\n",
-    "        elif o.tool_calls:\n",
-    "            yield SetNextNode(next_node=1)\n",
+    "        m = llm_stream.get_output()\n",
```

Comment:
if we call it "output", we should probably use "o" as the abbreviation and not "m"?

Reply:
Yep, artifact of the main branch merge, will fix